### PR TITLE
Docs: Docs for Rows to fields and Config from query transformations 

### PR DESCRIPTION
**Pull request by torkelo**
* [x] Add a list of all transforms to index page in alphabetical order
* [x] Remove duplicated info on the index page
* [x] Create new separate page for Rows to fields transform
* [x] Create new separate page for Config from query transform

### CODE REVIEW

File: docs/sources/panels/transformations/_index.md [hunk 1]
```diff
@@ -23,13 +22,23 @@ The order in which transformations are applied can make a huge difference in how
...
-  - [Logs]({{< relref "../visualizations/logs-panel.md" >}})
-  - [Stat]({{< relref "../visualizations/stat-panel.md" >}})
-  - [Table]({{< relref "../visualizations/table/_index.md" >}})
+- You must have entered a query that return data. For more information on queries, refer to [Queries]({{< relref "../queries.md" >}}).
+
+## List of transformations 
+
+- [Add field from calculation]({{< relref "./types-options.md#add-field-from-calculation" >}})
+- [Concatenate fields]({{< relref "./types-options.md#concatenate-fields" >}})
+- [Config from query results]({{< relref "./config-from-query.md" >}})
```
Comment (pkolyvas) [inline docs/sources/panels/transformations/_index.md:31]: @torkelo Should we start laying the foundation for our expanded "experimental" feature type by noting the status here? For example:

- [Config from query results]({{< relref "./config-from-query.md" >}}) *(Experimental)*

Comment (torkelo) [reply docs/sources/panels/transformations/_index.md:31]: Yea, not 100% these two transforms became a lot more solid than I originally expected. But could still be good to mark them as experimental, the threshold handling is especially so. thoughts @ryantxu ?

File: docs/sources/panels/transformations/config-from-query.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+### Options
+
+- **Config query**: Selet the query that returns the data you want to use as configuration.
+- **Apply to**: Select what fields or series to apply the configuration to.
+- **Apply to options**: Usually a field type or field name regex depending on what option you selected in **Apply to**.
+
+### Field mapping table
+
+Below the configuration listed above you will find the field table. Here all fields found in the data returned by the config query will be listed along with a **Use as** and **Select** option. This table gives you control over what field should be mapped to which config property and if there are multiple rows which value to select.
```
Comment (pkolyvas) [inline docs/sources/panels/transformations/config-from-query.md:20]: Suggestion: `Below the configuration listed above` is a little hard to parse. I tried to break it down into slightly smaller chunks.

What about: 

This transformation includes a field table which lists all fields in the data returned by the config query. This table gives you control over what field should be mapped to each config property. You can choose which value to select if there are multiple rows in the returned data. Each field has both a **Use as** and **Select** option.

I'm not sure if this is accurate or better now that I've written it. :/

**Review by pkolyvas — APPROVED** [review]
Left some comments, nothing blocking!

File: docs/sources/panels/transformations/config-from-query.md (hunk 2 above)
Comment (torkelo) [reply docs/sources/panels/transformations/config-from-query.md]: updated!

File: docs/sources/panels/transformations/config-from-query.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
++++
+title = "Config from query results"
+weight = 300
++++
+
+# Config from query results transform
```
[the remark this answers is not in the document]
Comment (torkelo) [reply docs/sources/panels/transformations/config-from-query.md:6]: Added a beta notice here. Still not sure this is experimental really, borderline :) And don't relly have time to introduce this new feature state & option for v8.1 (last day tomorrow before vacation). 

But would I would love to add here, and in the other transform AND in the in application help is a link to a Github discussion issue where we can collect feedback on this feature.
